clients/ui/frontend/.eslintrc

```diff
@@ -72,7 +72,6 @@
     "no-only-tests/no-only-tests": "error",
     "@typescript-eslint/default-param-last": "error",
     "@typescript-eslint/dot-notation": ["error", { "allowKeywords": true }],
-    "@typescript-eslint/lines-between-class-members": ["error", "always"],
```

Comment:
@Griffin-Sullivan https://eslint.org/blog/2023/10/deprecating-formatting-rules/

not sure if we should add something in Prettier to cover that.

Reply:
this is a rule I already didn't care for too much any way. For now we can just remove